download.md

```diff
@@ -4,9 +4,9 @@ title: Downloads
 permalink: /downloads/
 ---
 
-## PELUX 2.0
-As of 2018-07-02, PELUX 2.0 has been released! The docs can be found
-[here](//pelux.io/software-factory/PELUX-2.0/).
+## PELUX 3.0
+As of 2018-11-15, PELUX 3.0 has been released! The docs can be found
+[here](//pelux.io/software-factory/PELUX-3.0/).
 
 ### Main features
  - Derived from poky
```

Comment:
I'm guessing there will be more changes done to this page before the WIP flag is removed.